Scripts/DCS-BIOS/BIOS.lua

```diff
@@ -28,7 +28,9 @@ package.path = lfs.writedir() .. [[Scripts\DCS-BIOS\lib\modules\aircraft_modules
 package.path = lfs.writedir() .. [[Scripts\DCS-BIOS\lib\modules\common_modules\?.lua;]] .. package.path
 package.path = lfs.writedir() .. [[Scripts\DCS-BIOS\lib\modules\documentation\?.lua;]] .. package.path
 package.path = lfs.writedir() .. [[Scripts\DCS-BIOS\lib\modules\memory_map\?.lua;]] .. package.path
+package.path = lfs.writedir() .. [[Scripts\DCS-BIOS\lib\common\?.lua;]] .. package.path
 
+dofile(lfs.writedir()..[[Scripts\DCS-BIOS\lib\common\common.lua]])

```

Comment:
This line shouldn't be necessary as the previous line should cover it

Reply:
```package.path = lfs.writedir() .. [[Scripts\DCS-BIOS\lib\modules\aircraft_modules\?.lua;]] .. package.path```
This line?

Reply:
This one:
```lua
package.path = lfs.writedir() .. [[Scripts\DCS-BIOS\lib\common\?.lua;]] .. package.path
```

Reply:
```
package.path = lfs.writedir() .. [[Scripts\DCS-BIOS\lib\common\?.lua;]] .. package.path

dofile(lfs.writedir()..[[Scripts\DCS-BIOS\lib\common\common.lua]])
```
You mean that both of these are not needed.

Reply:
I removed 
```
package.path = lfs.writedir() .. [[Scripts\DCS-BIOS\lib\common\?.lua;]] .. package.path```